scope.py

```diff
@@ -1010,7 +1014,7 @@ def create_inference_script(
                 algorithm = 'xgb'
                 script.write('echo "xgb inference"\n')
                 for tag in model_tags:
-                    tag_file_gen = (group_path / tag).glob('*.h5')
+                    tag_file_gen = (group_path / tag).glob('*.json')
```

Comment:
Should this glob both?

Reply:
So far it seems XGB models are best saved as json files, and that's the only format our code saves them in. For now I think it's appropriate to only glob json extensions for the XGB algorithm.